include/lsst/afw/fits.h

```diff
@@ -510,6 +511,19 @@ class Fits {
 
 #endif // !SWIG
 
+//@{
+/// @brief Read FITS header
+///
+/// If 'strip' is true, common FITS keys that usually have non-metadata intepretations
+/// (e.g. NAXIS, BITPIX) will be ignored.
+///
+/// Includes support for the INHERIT convention: if 'INHERIT = T' is in the header, the
+/// PHU will be read as well, and nominated HDU will override any duplicated values.
+PTR(daf::base::PropertyList) readMetadata(std::string const & fileName, int hdu=0, bool strip=false);
```

Comment:
As I understand it, the `//@{` at the top of the doc and the `//@}` below the last declaration of `readMetadata` group the declarations together, so they all share the same doc.

The pointer is used because that's what we had before and I didn't want to change it without thinking more carefully.